packages/@react-spectrum/card/src/GridLayout.tsx

```diff
@@ -130,41 +130,6 @@ export class GridLayout<T> extends BaseLayout<T> {
     );
   }
 
-  getVisibleLayoutInfos(rect) {
```

Comment:
Handled by base class. Previously this implementation did not account for persisted keys.